### PR TITLE
List ZBit (ZBT)

### PR DESCRIPTION
**Pull request by ssl2**
- Official project URL: https://zbitofficial.io/
- Official block explorer URL: https://explorer.zbitofficial.io/

### CODE REVIEW

**Comment by yiowed** [comment]
Thanks @ManfredKarrer ! If you need anything, just let us know. We're in the forum too https://bisq.community/t/zbit-pull-request-added-to-repo/7273/2

File: assets/src/main/java/bisq/asset/coins/ZBit.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import bisq.asset.AltCoinAccountDisclaimer;
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+
+@AltCoinAccountDisclaimer("account.altcoin.popup.ZBT.msg")
+public class ZBit extends Coin {
+
+    public ZBit() {
+        super("ZBit", "ZBT", new DefaultAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/ZBit.java:28]: Do not use deprecated validator.

**Review by blabno — COMMENTED** [review]
NACK per comments

**Comment by blabno** [comment]
do not commit  `gradle/wrapper/gradle-wrapper.jar`

**Comment by ripcurlx** [comment]
@ssl2 Closing PR because of inactivity. Feel free to re-open when PR requirements are fulfilled.

**Pull request closed without merging**